tests/syntax_test_cshtml.cshtml

```diff
@@ -0,0 +1,210 @@
+<!-- SYNTAX TEST "Packages/HTML (C#)/razor.sublime-syntax" -->
+<!-- https://docs.microsoft.com/en-us/aspnet/core/mvc/views/razor?view=aspnetcore-2.1 -->
+@using Michael.Blyons.Example
```

Comment:
Cute, but let's do a generic `System.Collections.Generic` or something. 😉

Reply:
that certainly is a more real world namespace; somehow I had no imagination at the time, sorry! changed :)